zebrad/src/components/sync.rs

```diff
@@ -1160,7 +1156,7 @@ where
                 // https://github.com/ZcashFoundation/zebra/issues/2909
                 let err_str = format!("{e:?}");
                 if err_str.contains("AlreadyVerified")
-                    || err_str.contains("AlreadyInChain")
+                    || err_str.contains("AlreadyInState")
```

Comment:
Is this change necessary? Did you do a search and replace for all instances?

Reply:
> Is this change necessary?

Not strictly necessary, I'll split this rename out into a PR for checking the queues.

> Did you do a search and replace for all instances?

That's how I caught it 😎